lib/galaxy_test/api/test_vault.py

```diff
@@ -0,0 +1,99 @@
+import json
+import os
+from typing import Any, cast
+
+from requests import (
+    get,
+    put
+)
+
+from ._framework import ApiTestCase
+
+TEST_USER_EMAIL = "vault_test_user@bx.psu.edu"
+
+
+class VaultApiTestCase(ApiTestCase):
+
+    @classmethod
+    def handle_galaxy_config_kwds(cls, config):
```

Comment:
If you're using this method - you should be creating an integration test case. API test cases shouldn't require extra configuration.

Reply:
This is actually testing the user preferences api, which currently doesn't have coverage I believe. I'll rename `VaultApiTestCase` to `UserPreferencesApiTestCase` to make that clearer. However, if we don't override the `vault_config_file`, the default `NullVault` won't work. Is there a place to define defaults for an API test case, or is it just booting a vanilla Galaxy instance as I gathered?

Reply:
There's no problem moving that into an integration tests. API tests are supposed to run successfully against external servers without assuming any specific configuration.